etc/profile-m-z/steam.profile

```diff
@@ -36,6 +36,7 @@ noblacklist ${HOME}/.local/share/vpltd
 noblacklist ${HOME}/.local/share/vulkan
 noblacklist ${HOME}/.mbwarband
 noblacklist ${HOME}/.paradoxinteractive
+noblacklist ${HOME}/.paradoxlauncher
```

Comment:
Add it to `disable-programs.inc`

Reply:
Just to be sure, adding `blacklist ${HOME}/.paradoxlauncher` in `disable-programs.inc`, correct ?

Reply:
Yes.

And keep it sorted.

Reply:
Got it, thanks. It's done.